config/globals.js

```diff
@@ -1,5 +1,4 @@
 /* eslint-disable no-process-env */
-const { debug } = console;
```

Comment:
I would suggets using winston (src/logger) here, so the debug logs are hidden (unless LOG_LEVEL is set to debug)

Reply:
That would lead to a circular dependency as `src/logger/index.js` requires the globals.

Reply:
We could add a logger call with the config globals in `src/index.js` after the server start. I'm just not 100% sure whether the config should really be logged as it also contains secrets.

Reply:
@Parnswir könntest du bitte noch mal prüfen.

Reply:
IMO, the debug messages are suppressed anyway in all serious production setups, so I don't see the problem with secrets being exposed. This is primarily there to help visibility in the dev environment.

Reply:
I think there might a couple of different points here (thanks for staying on this 🙇):

1. Previously it was using `console.debug`, which doesn't care about any `LOG_LEVEL` settings I think, so you might already have some of those secrets showing up in process output/logs somewhere in your deployments? 🕵️ 
2. Just switching this to the `winston` logger here would introduce a circular dependency as the logger in turn reads the `LOG_LEVEL` from the `globals`. As suggested above, we could log from the `app.on('listening')` callback (which also seems like a more appropriate place), but my thinking was: Keep it simple and just don't create a risk of these value being logged. If someone is debugging, they're most likely interested in very specific values in certain places and can temporarily log them there, right? 💭 

This is my personal conclusion with the (maybe limited) knowledge I have. Double-checking point 1 seems somewhat important and in the scope of 2, I'm open to bring back the logging for development if you think the benefits outweigh the risks.

Thanks 🙂✌️

Reply:
A small addendum:

> console.debug: The `console.debug()` function is an alias for `console.log()`.

— https://nodejs.org/docs/latest-v12.x/api/console.html#console_console_debug_data_args

And:

> console.log: Prints to `stdout` with newline.

— https://nodejs.org/docs/latest-v12.x/api/console.html#console_console_log_data_args

So depending on how you handle `stdout` in your deployments, you might have to do some digging (and possibly change secrets after disabling these logs).

Reply:
👍 logging secrets is bad.